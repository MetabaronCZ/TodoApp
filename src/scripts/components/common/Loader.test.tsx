import React from 'react';

import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import { Loader } from 'components/common/Loader';
import { withMockedProviders } from 'test/component';

describe('components/common/Loader', () => {
  it('should render correctly', () => {
    const tree = render(withMockedProviders(<Loader />));
    expect(tree.container).toMatchSnapshot();
    expect(tree.getByText('â†»')).toBeInTheDocument();
  });
});
