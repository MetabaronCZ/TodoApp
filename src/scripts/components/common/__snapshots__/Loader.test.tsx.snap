// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`components/common/Loader > should render correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 20px 0;
}

.c1 {
  font-family: Open Sans,Arial,sans-serif;
  color: #222;
  font-size: 14px;
  line-height: 20px;
  width: 30px;
  height: 30px;
  font-size: 30px;
  line-height: 1;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  text-align: center;
  -webkit-animation: bqZzKE 1000ms linear infinite;
  animation: bqZzKE 1000ms linear infinite;
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      â†»
    </div>
  </div>
</div>
`;
